rustworkx-core/src/connectivity/isolates.rs

```diff
@@ -0,0 +1,144 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+use petgraph::visit::{IntoNeighborsDirected, IntoNodeIdentifiers, NodeIndexable};
+use petgraph::Direction::{Incoming, Outgoing};
+
+/// Return the fisolates in a graph object
```

Comment:
I think there's a small typo here 😅

Reply:
Ah good catch, thanks
```suggestion
/// Return the isolates in a graph object
```